testsuites/syncgateway/functional/tests/test_db_online_offline.py

```diff
@@ -474,13 +474,13 @@ def test_online_to_offline_changes_feed_controlled_close_longpoll(params_from_ba
         futures = dict()
         futures[executor.submit(seth.start_longpoll_changes_tracking, termination_doc_id=None)] = "polling"
         #  For windows, request is too slow, so have _offline request to start before creating docs
-        if sg_platform == "windows":
-            futures[executor.submit(sg_client.take_db_offline, cluster_conf, "db")] = "db_offline_task"
+        # if sg_platform == "windows":
```

Comment:
if no need to deal with windows condition, would you remove the code?

Reply:
Thanks for reminding. After testing out in jenkins, then I want to remove and forgot about it. updated now